src/Entity/FundingCycle.php

```diff
@@ -134,9 +132,6 @@ class FundingCycle extends Entity
      * @access protected
      *
      * @ORM\Column(type="date", nullable=true)
-     *
-     * @Assert\DateTime()
-     * @Assert\GreaterThan(propertyPath="startDate", message="End Date must be after Start Date")
```

Comment:
This constraint was requested in Pelagos-3841 back in Aug 2019. Do we need to recreate this logic?